StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1112ClosingParenthesisMustBeOnLineOfOpeningParenthesis.cs

```diff
@@ -24,7 +27,7 @@ public class SA1112ClosingParenthesisMustBeOnLineOfOpeningParenthesis : Diagnost
     {
         public const string DiagnosticId = "SA1112";
         internal const string Title = "Closing parenthesis must be on line of opening parenthesis";
-        internal const string MessageFormat = "TODO: Message format";
+        internal const string MessageFormat = "The closing parenthesis or bracket in a call to a C# method or indexer, or the declaration of a method or indexer, is not placed on the same line as the opening bracket when the element does not take any parameters.";
```

Comment:
This is a very long message to appear in the Errors window. I believe a better message format would be simply reusing the `Title`.